test/rules/completed-docs/defaults/tsconfig.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I don't want to extend because that adds unnecessary overhead. I just added `compilerOptions.target` to have some config in the file